browser-test/src/admin_program_creation.test.ts

```diff
@@ -34,6 +34,7 @@ describe('program creation', () => {
         '### This is an example of some static text with formatting',
     })
 
+    await page.waitForTimeout(100) // ms
```

Comment:
What's the timeout for?

Reply:
The test was being flaky.  The snapshot sometimes had time to load the API name preview and sometimes not.  This gives it time to ensure that our snapshot always has the API name preview.

Reply:
IIRC the selector timeout is >=1,000ms

Reply:
The selector will find the element, but the inner text will be blank.

Reply:
You could add an HTML class or empty element when the contents loads for the tests to look for

Reply:
Added the HTML class but the timing of the snapshot was still off.  I decided to remove the debounce because it felt laggy anyway.  Without the debounce, the test works fine.

Reply:
Makes sense, it's not a very expensive function so running it on every keystroke is likely fine.

Reply:
@bion Are you ok with this solution and me closing the conversation?

Reply:
It's not crossing the wire so removing the debounce is probably for the best anyway.